src/main/assembly/assembly.xml

```diff
@@ -17,6 +17,9 @@
 		<file>
 			<source>pluginctl.sh</source>
 		</file>
+		<file>
+			<source>gen_scriptdriver_script.sh</source>
```

Comment:
Note: If we move this file into the scripts directory, might need to update this reference

Reply:
This is a driver file . it has to be at root and hence no changes needed here.